### PR TITLE
Update how-to-watch-keys doc.

### PR DESCRIPTION
**Pull request by siyuanfoundation**
Part of doc-sprint:
https://github.com/etcd-io/website/issues/802

### CODE REVIEW

**Comment by siyuanfoundation** [comment]
cc @celestehorgan

File: content/en/docs/v3.6/tutorials/how-to-watch-keys.md [hunk 1]
```diff
@@ -4,9 +4,31 @@ description: Guide to watching etcd keys
...
+```
+-i, --interactive[=false]: interactive mode
+--prefix[=false]: watch on a prefix if prefix is set
+--rev=0: Revision to start watching
+--prev-kv[=false]: get the previous key-value pair before the event happens
+--progress-notify[=false]: get periodic watch progress notification from server
+```
+
+### Options inherited from parent commands
+```
```
Comment (jmhbnz) [inline content/en/docs/v3.6/tutorials/how-to-watch-keys.md:27]: nit: markdownlint wants fenced code blocks to have a language specified and be surrounded by blank lines

```suggestion

```bash
```

File: content/en/docs/v3.6/tutorials/how-to-watch-keys.md [hunk 2]
```diff
@@ -4,9 +4,31 @@ description: Guide to watching etcd keys
...
+* Install [`etcd` and `etcdctl`](https://etcd.io/docs/v3.6/install/)
+
+## Watching keys
 `watch` to get notified of future changes:
+```
+etcdctl watch $KEY [$END_KEY]
+```
+
+### Options
+```
```
Comment (jmhbnz) [inline content/en/docs/v3.6/tutorials/how-to-watch-keys.md:18]: ditto - markdownlint.

**Review by jmhbnz — APPROVED** [review]
Thanks @siyuanfoundation - overall looks good, couple of nits re markdown best practices.

File: content/en/docs/v3.6/tutorials/how-to-watch-keys.md (hunk 2 above)
Comment (siyuanfoundation) [reply content/en/docs/v3.6/tutorials/how-to-watch-keys.md]: done

**Pull request merged**